### DISCUSSION TITLE
Ссылка на полный список материалов

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/platform#387] [Идеи] [answered]
Сейчас, если список "зафичеренных" статей показан на главной не полностью, существует кнопка "Показать ещё". 

![Снимок экрана 2021-09-20 011302](https://user-images.githubusercontent.com/50330458/133944700-409ccf8e-005c-4fa3-b92d-bbdd9ca14562.png)

Когда статьи заканчиваются, кнопка просто пропадает. 

Не хотим ли мы вести пользователя на полный список всех статей? Например:

![Снимок экрана 2021-09-20 011352](https://user-images.githubusercontent.com/50330458/133944721-de024bcd-6d6f-40e1-9d7a-9552e59d40ad.png)

**Comment by nlopin** [comment]
Мне кажется, что вести на /all негуманно, там такая бесконечная портянка, что ничего не найти

**Reply by pepelsbey** [reply]
Поддержу, /all это совсем не то, что ты ожидаешь увидеть после красивого списка зафичеренных статей.

**Comment by nlopin** [comment, marked as answer]
Я предлагаю не делать ничего. После фичеринга идут разделы с крупной разбивкой, можно найти интересное там
